src/channel.ts

```diff
@@ -1249,6 +1249,12 @@ export class Channel<
       this.id = state.channel.id;
       this.cid = state.channel.cid;
       // set the channel as active...
+      const membersStr = Object.keys(this.state.members)?.sort().join(',');
+
+      if (`${this.type}:!members-${membersStr}` in this.getClient().activeChannels) {
+        delete this.getClient().activeChannels[membersStr];
```

Comment:
should we delete the first instance from the activeChannels?

Reply:
What do you mean? Its a map, so there will be only one instance.

But just noticed that we are not deleting the correct key from activeChannels. It should be something like:
```js
delete this.getClient().activeChannels[`${this.type}:!members-${membersStr}`];
```

Gonna add to test as well :)

Reply:
Done !!